float64[100] CylinderPosition            # 100 x 1
float64[100] LoadPinForces               # 100 x 1
float64[20] PressureA                   # 20 x 1
float64[20] PressureB                   # 20 x 1
float64[20] AngleXZ                     # 20 x 1
float64[20] AngleYZ                     # 20 x 1
ActuatorOrientation[100] CylinderDirection           # 100 x 3
float64[20] AxisPositionIst             # 20 x 1
float64[20] AxisPositionSetPoint        # 20 x 1
float64[20] AxisForceIst                # 20 x 1
float64[20] AxisForceSetPoint           # 20 x 1
float64[20] VCSetPoint                  # 20 x 1
float64[20] VCCurrentIstValue           # 20 x 1
float64[8] ADC_From328To335            # 8 x 1
DisplacementForceStruct[5] DisplacementForceB   # 5 x 1 struct
DisplacementForceStruct[5] DisplacementForceN   # 5 x 1 struct
DisplacementForceStruct[5] DisplacementForceT   # 5 x 1 struct
DisplacementForceStruct[5] DisplacementForceS   # 5 x 1 struct